docs/ImplicitMSALAuthenticationProvider.md

```diff
@@ -0,0 +1,54 @@
+#### Creating an instance of ImplicitMSALAuthenticationProvider in browser environment
+
+Refer devDependencies in [package.json](../package.json) for the compatible msal version and update that version in below.
+
+```html
+<script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
+```
+
+```typescript
+// Configuration options for MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/MSAL.js-1.0.0-api-release#configuration-options
+const msalConfig = {
+	auth: {
+		clientId: "your_client_id", // Client Id of the registered application
+		redirectUri: "your_redirect_uri",
+	},
+};
+const graphScopes = ["user.read", "mail.send"]; // An array of graph scopes
+
+// Important Note: This library implements loginPopup and acquireTokenPopup flow, remember this while initializing the msal
+// Initialize the MSAL @see https://github.com/AzureAD/microsoft-authentication-library-for-js#1-instantiate-the-useragentapplication
+const msalApplication = new Msal.UserAgentApplication(msalConfig);
+const options = new MicrosoftGraph.MSALAuthenticationProviderOptions(graphScopes);
+const authProvider = new MicrosoftGraph.ImplicitMSALAuthenticationProvider(msalApplication, options);
+```
+
+#### Creating an instance of ImplicitMSALAuthenticationProvider in node environment
+
+Refer devDependencies in [package.json](./package.json) for the compatible msal version and update that version in below.
+
+```cmd
+npm install msal@<version>
```

Comment:
```suggestion
npm install msal@<version> --save
```

Reply:
--save is no longer needed. 
https://stackoverflow.com/questions/19578796/what-is-the-save-option-for-npm-install

Reply:
oh I didn't know. Thanks, I guess I'm old :)